### PR TITLE
Remove redundant command line options

### PR DESCRIPTION
**Pull request by Naviabheeman**
According to #216 options enablebip61, permitbaremultisig and rpcserialversion have been removed in this PR

### CODE REVIEW

File: test/functional/p2p_invalid_tx.py [hunk 1]
```diff
@@ -157,11 +157,11 @@ def run_test(self):
 
         # restart node with sending BIP61 messages disabled, check that it disconnects without sending the reject message
         self.log.info('Test a transaction that is rejected, with BIP61 disabled')
-        self.restart_node(0, ['-enablebip61=0','-persistmempool=0'])
+        self.restart_node(0, ['-persistmempool=0'])
         self.reconnect_p2p(num_connections=1)
         node.p2p.send_txs_and_test([tx1], node, success=False, expect_disconnect=False)
         # send_txs_and_test will have waited for disconnect, so we can safely check that no reject has been received
-        assert_equal(node.p2p.reject_code_received, None)
+        assert_equal(node.p2p.reject_code_received, 64)
```
Comment (azuchi) [inline test/functional/p2p_invalid_tx.py:164]: What kind of error is 64?

File: test/functional/p2p_invalid_tx.py [hunk 2]
```diff
@@ -157,11 +157,11 @@ def run_test(self):
 
         # restart node with sending BIP61 messages disabled, check that it disconnects without sending the reject message
         self.log.info('Test a transaction that is rejected, with BIP61 disabled')
```
Comment (azuchi) [inline test/functional/p2p_invalid_tx.py:159]: > self.log.info('Test a transaction that is rejected, with BIP61 disabled')

Is it possible to disable BIP61?

**Review by azuchi — CHANGES_REQUESTED** [review]
The removed parameters are still described in `doc/man/tapyrusd.1`.

File: test/functional/p2p_invalid_tx.py (hunk 2 above)
Comment (Naviabheeman) [reply test/functional/p2p_invalid_tx.py]: no. I removed the comment and changed the test.

File: test/functional/p2p_invalid_tx.py (hunk 1 above)
Comment (Naviabheeman) [reply test/functional/p2p_invalid_tx.py]: it was non-standard tx. but strangely its not happening anymore. I'll wait for the CI and check further

File: test/functional/p2p_invalid_tx.py [hunk 3]
```diff
@@ -20,8 +20,9 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
-    wait_until,
+    wait_until
 )
+from time import sleep
```
Comment (azuchi) [inline test/functional/p2p_invalid_tx.py:25]: This feature does not appear to be used.

**Review by azuchi — CHANGES_REQUESTED** [review]

**Review by azuchi — APPROVED** [review]

**Pull request merged**